python/huggingfaceserver/README.md

```diff
@@ -111,8 +111,8 @@ spec:
           cpu: 100m
           memory: 2Gi
 ```
-3. Serve the huggingface model using vllm runtime. vllm is the default runtime. Note - Model need to be supported by vllm otherwise KServe python runtime will be used as a failsafe.
-vllm supported models - https://docs.vllm.ai/en/latest/models/supported_models.html 
+3. Serve the llama2 model using huggingface vLLM runtime. For the llama2 model, vLLM is supported and used as the default model. If available for a model, vLLM is set as the default runtime. Note - Model needs to be backed by vLLM otherwise KServe python runtime will be used as a failsafe.
```

Comment:
using `HuggingFace vLLM runtime`
Is it the same runtime as `KServe python HuggingFace runtime`? If so, we can use the same name; If not, what is the difference?